api/spec/spec_handler.go

```diff
@@ -147,6 +147,7 @@ var (
 	mountOptionsRegex             = regexp.MustCompile(api.SpecMountOptions + `=([A-Za-z0-9:;@=#]+),?`)
 	sharedv4MountOptionsRegex     = regexp.MustCompile(api.SpecSharedv4MountOptions + `=([A-Za-z0-9:;@=#]+),?`)
 	cowOnDemandRegex              = regexp.MustCompile(api.SpecCowOnDemand + "=([A-Za-z]+),?")
+	checksumDataRegex             = regexp.MustCompile(api.SpecChecksumData + "=([A-Za-z]+),?")
```

Comment:
is this regex correct?
Can you not give checksum_data=0|1?
parsebool will allow it.

Reply:
Yes, parsebool allows it, fixed it.